apps/koku-ui-onprem/Dockerfile

```
@@ -0,0 +1,21 @@
+FROM node:20-alpine AS builder
+WORKDIR /app
+
+COPY package.json ./
+COPY package-lock.json ./
+COPY tsconfig.json ./
+COPY apps/koku-ui-onprem/package.json ./apps/koku-ui-onprem/
+COPY libs ./libs
+RUN npm ci
+
+COPY apps/koku-ui-onprem ./apps/koku-ui-onprem/
+
+RUN npm run build --workspace=@koku-ui/koku-ui-onprem
+
+FROM registry.redhat.io/rhel10/nginx-126
```

Should we use UBI instead of rhel ?
```suggestion
FROM registry.redhat.io/ubi10/nginx-126
```